website/www/site/data/capability_matrix.yaml

```diff
@@ -36,1463 +36,1463 @@ capability-matrix:
   categories:
     - description: What is being computed?
       anchor: what
-      color-y: "fff"
-      color-yb: "f6f6f6"
-      color-p: "f9f9f9"
-      color-pb: "d8d8d8"
-      color-n: "e1e0e0"
-      color-nb: "bcbcbc"
+      color-y: 'fff'
```

Comment:
Is there a reason changing " -> '? They look redundant

Reply:
I think, it's due to my vscode setup.